src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  padding: 0 50px;
+}
+
+.nav-list {
+  list-style-type: none;
+  display: flex;
+  margin: 0;
+  padding-left: 0;
+}
+
+.nav-link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  color: black;
+  text-decoration: none;
+}
+
+.nav-item {
+  cursor: default;
+  justify-content: center;
+}
+
+.is-active {
+  margin-left: 0;
+  color: var(--active-color);
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: var(--active-color);
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+:hover {
+  color: var(--active-color);
+}
```

Comment:
The `:hover` selector is too generic and will apply to all elements. It is recommended to specify the element or class it should apply to, such as `.nav-link:hover`, to avoid unintended styling.